packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -52,20 +55,25 @@ const insertBandwidthLimit = (sdpLines, index) => {
 
 /**
  * needed for calliope max-fs
- * @param {String} sdp 
+ * @param {String} sdp
+ * @param {String} [level=QUALITY_LEVELS.HIGH] quality level for max-fs
  * @returns {String}
  */
-const setMaxFs = (sdp) => {
+const setMaxFs = (sdp, level = QUALITY_LEVELS.HIGH) => {
+  if (!MAX_FRAMESIZES[level]) {
+    throw new ParameterError(`setMaxFs: unable to set max framesize, value for level "${level}" is not defined`)
+  }
   // eslint-disable-next-line no-warning-comments
   // TODO convert with sdp parser, no munging
   let replaceSdp = sdp;
-  replaceSdp = replaceSdp.replace(/(\na=fmtp:(\d+).*level-asymmetry-allowed=1.*)/gi, `$1;${SDP.MAX_FS_8160}`)
+  const maxFsLine = `${SDP.MAX_FS}${MAX_FRAMESIZES[level]}`
+  replaceSdp = replaceSdp.replace(/(\na=fmtp:(\d+).*level-asymmetry-allowed=1.*)/gi, `$1;${maxFsLine}`)
```

Comment:
Hopefully we'll get some bandwidth to begin implementing SDP parsers to make areas like this read easier.